### PR TITLE
Fix patched versions for ruby 2.6.0

### PR DESCRIPTION
**Pull request by wintersolutions**
According to:

* https://www.ruby-lang.org/en/news/2018/10/17/openssl-x509-name-equality-check-does-not-work-correctly-cve-2018-16395/
* https://www.ruby-lang.org/en/news/2018/10/17/not-propagated-taint-flag-in-some-formats-of-pack-cve-2018-16396/

CVE-2018-16395 and CVE-2018-16396 only affect `2.6.0-preview2` and earlier.

### CODE REVIEW

**Comment by vipulnsward** [comment]
Would love to get this in, 2.6 usage fails without it.

Example: https://circleci.com/gh/bigbinary/wheel/1038

File: rubies/ruby/CVE-2018-16396.yml [hunk 1]
```diff
@@ -23,3 +23,5 @@ patched_versions:
   - "~> 2.3.8"
   - "~> 2.4.5"
   - "~> 2.5.2"
+  - "~> 2.6.0-preview3"
+
```
Comment (JonRowe) [inline rubies/ruby/CVE-2018-16396.yml]: Extra blank line here!

**Comment by wintersolutions** [comment]
Removed extra blank line.

**Comment by wintersolutions** [comment]
@JonRowe Yeah I'm using rubymine with the setting add blank endline.

**Comment by wintersolutions** [comment]
If this is breaking your CI with `ruby-audit` you can run:

`ruby-audit -i CVE-2018-16395 CVE-2018-16396`

as a workaround.

**Comment by kaspergrubbe** [comment]
Any chance of getting this merged so we don't receive false positives?

**Comment by kaspergrubbe** [comment]
@JonRowe Ping.

**Comment by JonRowe** [comment]
Sorry I was a drive by reviewer, I don't have rights!

**Comment by kaspergrubbe** [comment]
I'll let you off then!

**Comment by phillmv** [comment]
Thanks for the PR!

**Pull request merged**